app/exchanges/exchange_provider.py

```diff
@@ -0,0 +1,30 @@
+from typing import Dict
+
+from enums.exchange import Exchange
+from exchanges.base_exchange import ExchangeAPI
```

Comment:
This can make import more readable and scalable by changing `__init__.py` which causing import to be shorten to:
```python
from exchanges import (BinanceAPI, BitkubAPI, FtxAPI, ...)
```